### ISSUE TITLE
filter query on qualities gives unintuitive result [or maybe I give up on using filters/the website?]

### ISSUE THREAD

**Issue by amueller** [openml/OpenML#889]
Running this filter:
https://www.openml.org/search?q=qualities.MajorityClassPercentage%3E99&type=data

gives me some arbitrary datasets. I don't know why.

**Comment by joaquinvanschoren** [comment]
you were missing the colon ':'
qualities.MajorityClassPercentage:>99
https://www.openml.org/search?q=qualities.MajorityClassPercentage%3A%3E99&type=data

**Comment by amueller** [comment]
you're right. So then it just became a search query instead of a condition?
